packages/core/src/web/app/components/AiGenerate/components/StickyFooter.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Button } from 'antd';
+
+import FluxIcons from '@core/app/icons/flux/FluxIcons';
+import useI18n from '@core/helpers/useI18n';
+import browser from '@core/implementations/browser';
+import type { IUser } from '@core/interfaces/IUser';
+
+import styles from '../index.module.scss';
```

Comment:
nitpick:
If the classnames are only used in StickyFooter, create a StickyFooter for it